packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -649,6 +649,7 @@ export const BaseInput = React.forwardRef<HTMLInputElement, BaseInputProps>(
           justifyContent={isLabelLeftPositioned ? 'center' : undefined}
           alignItems={isLabelLeftPositioned ? 'center' : undefined}
           position="relative"
+          width="100%"
```

Comment:
why this?

Reply:
Without this, the input will break if we change the display property